src/components/TweetForm.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+
+const TweetForm = ({ addTweet, message, setMessage, maxChars }) => {
+  const [userName, setUserName] = useState("Brittany");
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    if (message.length > maxChars) {
+      alert("Tweet exceeds 120 characters!");
+      return;
+    }
+
+    if (userName && message) {
+      const newTweet = {
+        id: Date.now(),
+        userName,
+        text: message,
+        date: new Date().toLocaleString(),
```

Comment:
Can you explain to me the difference between the` now()` and `toLocaleString()` functions? One requires creating a `new Date() `and the other doesn't. Why is this? (I'm looking for an OOP explanation.)

Reply:
Oh wow! I didn't know that `Date.now()` returns the number of milliseconds since midnight, January 1, 1970! `new Date()` creates an object and returns the current date and time. `.toLocaleString` formats the date with numbers (ex 12/16/2023 3:02:?? PM).